src/Microsoft.Azure.SignalR.Common/Exceptions/AzureSignalRAccessTokenNotAuthorizedException.cs

```diff
@@ -1,11 +1,14 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System;
+
 namespace Microsoft.Azure.SignalR.Common
 {
     /// <summary>
     /// The exception throws when AccessKey is not authorized.
     /// </summary>
+    [Obsolete("Use AzureSignalRCredentialUnauthorizedException instead.")]
```

Comment:
why? is using this exception acceptable? This is a breaking change?

Reply:
Use the same exception but add a new constructor.